src/js/features/arbitration/actions.js

```diff
@@ -29,7 +29,7 @@ import EscrowProxy from '../../../embarkArtifacts/contracts/EscrowProxy';
 ArbitrationLicense.options.address = ArbitrationLicenseProxy.options.address;
 Escrow.options.address = EscrowProxy.options.address;
 
-export const getDisputedEscrows = () => ({type: GET_DISPUTED_ESCROWS});
+export const getDisputedEscrows = (fallbackDisputes = false) => ({type: GET_DISPUTED_ESCROWS, fallbackDisputes});
```

Comment:
The flag name is not very clear. It took me a while to get what it meant. It should probably be `inludeFallbackDisputes`